lighthouse-core/gather/gatherers/trace-elements.js

```diff
@@ -82,12 +82,20 @@ class TraceElements extends Gatherer {
     const shiftEvents = mainThreadEvents
       .filter(e => e.name === 'LayoutShift')
       .map(e => e.args && e.args.data);
+    const indexFirstEventWithoutInput =
+      shiftEvents.findIndex(event => event && !event.had_recent_input);
 
-    shiftEvents.forEach(event => {
-      if (!event || !event.impacted_nodes || !event.score || event.had_recent_input) {
+    shiftEvents.forEach((event, index) => {
+      if (!event || !event.impacted_nodes || !event.score) {
         return;
       }
 
+      // Ignore events with input, unless it's one of the initial events.
+      // See comment in computed/metrics/cumulative-layout-shift.js.
+      if (indexFirstEventWithoutInput !== -1 && index >= indexFirstEventWithoutInput) {
```

Comment:
I had to think through this -1 case for quite a bit, I wish it were easier to default it back to `Infinity`/choose your magical "not found" number